samples/fan_in_fan_out/HttpStart/__init__.py

```diff
@@ -1,12 +1,12 @@
 import logging
-
+import json
 import azure.functions as func
 import azure.durable_functions as df
 
 
 async def main(req: func.HttpRequest, starter: str) -> func.HttpResponse:
     client = df.DurableOrchestrationClient(starter)
-    payload = req.get_body()
+    payload: str = json.loads(req.get_body().decode())
```

Comment:
What exactly is this change for?

It may be worth commenting why this is necessary, especially since this is a sample.

Reply:
Ah yes, this is needed to load the JSON HTTP payload, which is otherwise generally interpreted as bytes. Added the comment!